src/superqModule.cpp

```diff
@@ -562,7 +562,7 @@ bool SuperqModule::close()
      if (!portSuperq.isClosed())
         portSuperq.close();
 
-    if (mode_online)
+    if (mode_online && visualization_on)
```

Comment:
Small addition: avoid connecting to the gaze controller when we specify that we don't want to display the sample points representing the superquadric.